doc/advanced-configuration/Listener-modules.md

```diff
@@ -180,6 +180,7 @@ Interface for XMPP components ([XEP-0114: Jabber Component Protocol](http://xmpp
 * `service_check_from` (boolean, default: `true`) - Checks whether the server should verify the "from" field in stanzas from the component
 * `max_fsm_queue` (positive integer, the value of this option set global) - message queue limit to prevent resource exhaustion; overrides the global value of this option
 * `hidden_components` (boolean, default: `false`) - All components connected to an endpoint with this option enabled will be considered "hidden" (see explanation below).
+* `conflict_behaviour` (`disconnect`, `kick_old`, default: `disconnect`) - Stop previous connection, if there is a routing conflict. (see explanation below).
```

Comment:
Stops the previous connection, if there is a routing conflict (see explanation below).